src/Form.js

```diff
@@ -2,6 +2,7 @@
 
 const Form = ({ handleInputChange, handleSubmit}) => {
 
+  
 
   return(
   <form action="submit">
```

Comment:
I also had my form send data over with a button using onClick, but I was told it should be a onSubmit on the form. It's semantics, but hey now I can make a comment

Reply:
oh lovely thank you